codegen/src/generator/xml_payload_parser.rs

```diff
@@ -301,12 +301,9 @@ fn generate_struct_deserializer(name: &str, service: &Service, shape: &Shape) ->
         return format!(
             "try!(start_element(tag_name, stack));
 
-            stack.next();
```

Comment:
Note that I had to delete these to get the tests to pass. These appear to be the only responses that have wrapper tags that also have empty members:

```sh
$ grep -R '/>' botocore/tests/unit/response_parsing/xml/responses/* | grep Result
botocore/tests/unit/response_parsing/xml/responses/ses-delete-identity.xml:  <DeleteIdentityResult/>
botocore/tests/unit/response_parsing/xml/responses/ses-set-identity-dkim-enabled.xml <SetIdentityDkimEnabledResult/>
```
The current logic was wrong, and changing this to not consume the stack doesn't appear to have broken anything.